### DISCUSSION TITLE
error This portal is currently experiencing database issues. Please report this to the portal administrator. Thank you for your help.

### DISCUSSION THREAD

**Discussion by JuanDavRH** [chamilo/chamilo-lms#5153] [General] [answered]
Buen dia, 

Alguno de casualidad sabe la solución al error This portal is currently experiencing database issues. Please report this to the portal administrator. Thank you for your help. 

Contexto:
Se esta intentando actualizar chamilo, se reemplazaron los archivos de la antigua versión la cual es 1.11.12 con los de la nueva la cual es 1.11.26, la base de datos esta funcional. 

Se instalo php7.4 y la versión de mysql es la 8

Seria de gran ayuda cualquier información que me puedan proporcionar.

**Comment by ywarnier** [comment, marked as answer]
Hola @JuanDavRH 
He movido tu pregunta a nuestro nuevo espacio de foro, porque los "issues" son para errores del software.
Aquí te enfrentas a un error de disponibilidad o de estructura de base de datos.

Si eso sucede de manera permanente después de haber actualizado, puede ser que te falte limpiar el caché de la versión anterior. Para esto, puedes simplemente hacer un borrado de todo el contenido de app/cache/ y asegurarte que los permisos en app/cache/ siguen permitiendo al servidor web escribir en esta carpeta. Es probable que resuelva el problema. Darle una mirada al error_log de tu servidor web siempre ayuda.

Si a cambio el error aparece ocasionalmente, es probable que tu servidor de bases de datos esté simplemente algo sobrecargado. Deberías poder observarlo en reportes de algun tipo sobre la carga de tu servidor de bases de datos. En este caso, hay que aumentar los recursos del servidor de bases de datos, o aumentar sus límites de simultaneidad de conexiones. Esto ya es fuera de temas de Chamilo.
